fili-system-config/src/main/java/com/yahoo/bard/webservice/config/LayeredFileSystemConfig.java

```diff
@@ -64,11 +64,6 @@
     private final Properties runtimeProperties;
 
     /**
-     * Loader pulls resources in from class path locations.
-     */
-    private final ConfigResourceLoader loader = new ConfigResourceLoader();
```

Comment:
Right, I'm just moving it to be a local variable, since it's only used in 1 method.